libsql/src/hrana/hyper.rs

```diff
@@ -85,7 +85,11 @@ impl HttpSend for HttpSender {
     }
 
     fn oneshot(self, url: Arc<str>, auth: Arc<str>, body: String) {
-        tokio::spawn(self.send(url, auth, body));
+        if let Ok(rt) = tokio::runtime::Handle::try_current() {
+            rt.spawn(self.send(url, auth, body));
+        } else {
+            tracing::warn!("tried to send request to `{url}` while no runtime was available");
+        }
```

Comment:
in what situation wouldn't there be a runtime?

Reply:
At some point `execute_transaction` suggested that by panicking from inside of that method.